cms/admin.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+session_start();
+
+if ($_SESSION['loggedIn'] !== true) {
+    header('Location: instructions.php');
+}
+
+
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Content Management System</title>
+</head>
+<body>
+<nav>
+    <ul>
+        <li><a href="portfolio.php">Portfolio</a></li>
```

Comment:
what does this page do? if it is just a link to another page then delete this page and link there

Reply:
This page used to host navigation links for multiple functionalities,  most of it removed due to MVP, will still built up on it as a navigator for further CMS functionality.